### ISSUE TITLE
write_spss should provide set variability

### ISSUE THREAD

**Issue by jamesrkg** [Quantipy/quantipy#37]
``write_spss`` is currently hard-coded to use the ``'data file'`` set when finding columns for export. This should instead be the default value of a new parameter, allowing users to specify any set instead.

The docs example for ``read_spss`` is incorrect as well. Currently it says:

```python
>>> from quantipy.core.tools.dp.io import read_spss
>>> meta, data = read_spss(path_sav, meta, data)
```

But it should say:

```python
>>> from quantipy.core.tools.dp.io import read_spss
>>> meta, data = read_spss(path_sav, **kwargs)
```

**Comment by jamesrkg** [comment]
This will be resolved by #61

**Comment by jamesrkg** [comment]
Resolved.
